src/main/java/br/com/github/zsguil/maquina_cafe/classes/ColecaoBebidas.java

```diff
@@ -0,0 +1,42 @@
+package br.com.github.zsguil.maquina_cafe.classes;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import br.com.github.zsguil.maquina_cafe.classes.abstracts.Bebida;
+import br.com.github.zsguil.maquina_cafe.classes.abstracts.SistemaInterno;
+import br.com.github.zsguil.maquina_cafe.classes.bebidas.AguaQuente;
+import br.com.github.zsguil.maquina_cafe.classes.bebidas.Cafe;
+import br.com.github.zsguil.maquina_cafe.classes.bebidas.CafeLeite;
+import br.com.github.zsguil.maquina_cafe.classes.bebidas.Cappucino;
+import br.com.github.zsguil.maquina_cafe.classes.bebidas.Cha;
+
+public class ColecaoBebidas extends SistemaInterno {
+	private Map<Integer, Bebida> bebidas = new LinkedHashMap<>();
+	
+	public void adicionaItens() {
+		Bebida[] arrayBebidas = 
+			{new Cafe(), new CafeLeite(), new Cappucino(), new Cha(),new AguaQuente()};
+		
+		int key = 0;
+		
+		for(Bebida bebiba : arrayBebidas) {
```

Comment:
Você poderia já ter criado o Map de bebidas inicializado com os valores, daí não ia precisar desse for aqui. Tipo:

`private Map<Integer, Bebida> bebidas = Map.of(0, new Cafe(), 1, new CafeLeite())`